include/sound/sof/ipc4/header.h

```diff
@@ -108,6 +108,11 @@ enum sof_ipc4_global_msg {
 
 	/* Loads library (using Code Load or HD/A Host Output DMA) */
 	SOF_IPC4_GLB_LOAD_LIBRARY,
+	/*
```

Comment:
typos in commit message
"it's" 
"recceives"

Reply:
OK, rewording

Reply:
nit-pick: DSP side sets it's buffer -> DSP side sets its buffer